README.md

```diff
@@ -27,6 +27,20 @@ that creates additional scrape targets. Each scrape target is expected
 to expose a `/metrics` HTTP path that exposes its metrics in a
 Prometheus compatible format.
 
+For example if it desired to scrape metrics regarding the state of
+objects in a Kubernetes cluster, the
```

Comment:
For example, the [kube-state-metrics](https://charmhub.io/kube-state-metrics) charm interoperates with the Prometheus K8S charm in a way that allows you import metrics about resources in a Kubernetes by doing:

<cli commands>

Reply:
Ack. Will change.